### PR TITLE
New hCNV galaxy tutorial 

### PR DESCRIPTION
**Pull request by khaled196**

### CODE REVIEW

**Comment by khaled196** [comment]
The tutorial is available in topics/variant analysis/tutorials/somatic-variants-discovery/tutorial.md 

I added the workflow and images for the tutorial

I also updated the counterparts yml file

File: topics/variant-analysis/tutorials/somatic-variant-discovery/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,725 @@
...
+time_estimation: 3H
+key_points:
+- Follow best practices for read mapping, quality control and mapped reads postprocessing
+  to minimize false-positive hCNVs.
+contributors:
+- Khaled Jum'ah
+- Katarzyna Kamieniecka
+- Wolfgang Maier
+- David Salgado
+- Krzysztof Poterlowicz
```
Comment (hexylena) [inline topics/variant-analysis/tutorials/somatic-variant-discovery/tutorial.md:23]: These need to be usernames, not the human names, just as they are in CONTRIBUTORS.yaml :) 

Also, if you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing/testing/etc.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.

File: topics/variant-analysis/tutorials/somatic-variant-discovery/workflows/Control-FREEC_MIRACUM_-_somatic_variant_discovery_from_WES_data.ga [hunk 2]
```diff
@@ -0,0 +1,1402 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "",
```
Comment (hexylena) [inline topics/variant-analysis/tutorials/somatic-variant-discovery/workflows/Control-FREEC_MIRACUM_-_somatic_variant_discovery_from_WES_data.ga:3]: I see the following errors for the workflow, could you add tags and an annotation for it?

```
Warning:  The tags attribute is missing. Please add: `"tags": ["variant-analysis"]`
Warning:  The 'annotation' attribute is missing or incorrect. Please add: `"annotation": "<title of tutorial>"`
make: *** [Makefile:124: check-workflows] Error 123
```

**Review by hexylena — COMMENTED** [review]
Thanks for fixing the issues so quickly @khaled196! Here are a couple more, now that I notice the automatic checks.

**Comment by khaled196** [comment]
Hi Helena, 

Thank you for your comments, kindly find the changes on the contributors, workflow and the tutorial as proposed

**Review by hexylena — APPROVED** [review]

**Comment by hexylena** [comment]
Thank you for all of the fixes @khaled196 ! Great to have this in here :) I think we could possibly improve the circos plot a bit more, but maybe that's something for another time.

**Pull request merged**